src/Service/PHPEncryptionAdapter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SilverStripe\MFA\Service;
+
+use Defuse\Crypto\Crypto;
+use Defuse\Crypto\Exception\CryptoException;
+use SilverStripe\MFA\Exception\EncryptionAdapterException;
+
+/**
+ * An encryption adapter for defuse/php-encryption, enabled by default.
+ */
+class PHPEncryptionAdapter implements EncryptionAdapterInterface
```

Comment:
Should this mention `Defuse`? `PHPEncryption` is a little generic.

Reply:
php-encryption is the package name, which is why I named it this way, but I do agree with you. I'll call it `DefusePHPEncryptionAdapter`